src/foam/java/refinements.js

```diff
@@ -434,8 +433,11 @@ foam.CLASS({
       var info = this.SUPER(cls);
       var m = info.getMethod('cast');
       m.body = 'return ( o instanceof Number ) ?'
-            + '((Number)o).intValue() :'
-            + '(int)o;';
+        + ' ((Number)o).intValue() :'
```

Comment:
Would be nicer as a `-quoted multi-line string. Other similar code blocks below as well.